### PR TITLE
Bambenek connections

### PR DESCRIPTION
**Pull request by orhovy**
Added support in the csvApiModule for relationships.
Added Generic creation of relationship in the csvApiModule
Added data in the conf of Bambenek feed for the relation.
Added a new param in Bambenek feed for whether to create relations.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34688
blocked by: https://github.com/demisto/content/pull/12098

## Description
Added support in the csvApiModule for relationships.
Added Generic creation of relationship in the csvApiModule
Added data in the conf of Bambenek feed for the relation.
Added a new param in Bambenek feed for whether to create relations.

## Screenshots

https://user-images.githubusercontent.com/58938354/115690923-38ceca00-a366-11eb-93bc-b1ef8f6342ed.mov

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.py [hunk 1]
```diff
@@ -124,5 +149,5 @@ def main():
 
 from CSVFeedApiModule import *  # noqa: E402
 
-if __name__ == '__builtin__' or __name__ == 'builtins':
+if __name__ == '__builtin__' or __name__ == 'builtins' or __name__ == '__main__':
```
Comment (bakatzir) [inline Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.py:152]:
```suggestion
if __name__ in ('__builtin__', 'builtins', '__main__'):
```

**Review by bakatzir — COMMENTED** [review]
Looks very clean, just UT afterwards.

File: Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py [hunk 2]
```diff
@@ -309,11 +311,24 @@ def fetch_indicators_command(client: Client, default_indicator_type: str, auto_d
...
+                            )
+                            relationships_of_indicator = [relationships_lst.to_indicator()]
+
                     indicator = {
                         'value': value,
                         'type': indicator_type,
                         'rawJSON': raw_json,
-                        'fields': fields_mapping
+                        'fields': fields_mapping,
+                        'relationships': relationships_of_indicator
```
Comment (bakatzir) [inline Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py:331]:
```suggestion
                        'relationships': relationships_of_indicator,
```

**Review by bakatzir — COMMENTED** [review]
Good work. let's just have the UT have actual values  with accordance to the entity type, IPs in this case.

**Review by yuvalbenshalom — APPROVED** [review]

**Review by bakatzir — APPROVED** [review]

**Pull request merged**